common/services/contentful/contentful.ts

```diff
@@ -142,20 +143,32 @@ export class ContentfulService {
     })
   }
 
+  // TO DO this needs caching in redis 
   async fetchEntries() {
-    const entries = (await this.client.getEntries({
-      content_type: this.contentType,
-    })) as contentful.EntryCollection<ContentfulFields>
-
-    if (!entries.items?.length) {
-      return []
+    let cachedEntries = await get(`cache:entries:${this.contentType}`, true)
+
+    if(cachedEntries) {
+      return cachedEntries
+    } else {
+      const entries = (await this.client.getEntries({
+        content_type: this.contentType,
+      })) as contentful.EntryCollection<ContentfulFields>
+
+      if (!entries.items?.length) {
+        return []
```

Comment:
```suggestion
        entries = []
```

If we return here, we're not setting anything in the cache and then we will still be making lots of calls when there are no entries